### DISCUSSION TITLE
Typo in Copilot FAQ

### DISCUSSION THREAD

**Discussion by lewisfm** [community/community#19127] [Copilot Conversations] [answered]
Looks like someone tried to make a Markdown link but hit `)` instead of `]`.
<https://github.com/features/copilot#how-can-i-control-the-use-of-my-data-collected-by-copilot>
```diff
- [adjusting your user settings)
+ [adjusting your user settings]
```

![image](https://user-images.githubusercontent.com/75394401/174941546-0acc0ef4-b578-47ef-bdaa-0632dec7d2f3.png)

**Comment by melvincwng** [comment, marked as answer]
Hi @doinkythederp, thanks for highlighting this to the community.

For bugs, please kindly raise a ticket and report it to [Github Bug Report](https://support.github.com/contact/bug-report).

They usually will respond within a couple of days, after investigating your issue.

Hope this clarifies.

**Reply by melvincwng** [reply]
@doinkythederp, if this reply has helped you, please kindly 'Mark as answer' so that other people are informed that this issue has been addressed.

Thank you and have a nice day ahead 😊
